src/tests/dotnet-counters/JSONExporterTests.cs

```diff
@@ -133,6 +136,70 @@ public void ValidJSONFormatTest()
                 Assert.EndsWith("0 }]}", json);
             }
         }
+
+        [Fact]
+        public void TagsTest()
```

Comment:
Can tags have values that need to be escaped in json? (Like a string or something like an object)

Reply:
It wouldn't be good practice but technically yes they could, good call.